ocaml/testsuite/tests/typing-layouts-vec128/basics_beta.ml

```diff
@@ -0,0 +1,778 @@
+(* TEST
+ flags = "-extension layouts_beta -extension simd_beta";
```

Comment:
Currently the only difference is that alpha enables `or_null`, so the error messages print `value_or_null` instead of `value` in several places. Is that worth keeping?

Reply:
Aha, that's fine, we can combine them after that is changed